pkg/server/helpers.go

```diff
@@ -404,3 +405,20 @@ func (c *criContainerdService) ensureImageExists(ctx context.Context, ref string
 	}
 	return &newImage, nil
 }
+
+// resolveSymbolicLink resolves a possbile symlink path. If the path is a symlink, returns resolved
+// path and true; if the path is not a symlink, returns original path and false.
+func resolveSymbolicLink(path string) (string, bool, error) {
+	info, err := os.Lstat(path)
+	if err != nil {
+		return "", false, err
+	}
+	if info.Mode()&os.ModeSymlink != os.ModeSymlink {
+		return path, false, nil
+	}
+	resolved, err := filepath.EvalSymlinks(path)
```

Comment:
```
If path is relative the result will be relative to the current directory, unless one of the components is an absolute symbolic link. EvalSymlinks calls Clean on the result.
```
https://golang.org/pkg/path/filepath/#EvalSymlinks

It seems that the function keeps relative/absolute.